app/controllers/pokemon_controller.rb

```diff
@@ -39,6 +41,14 @@ def update
     end 
   end
 
+  def destroy
+    @pokemon = Pokemon.find(params[:id])
+    pokemon_name = @pokemon.nickname
```

Comment:
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔

(You can still call `@pokemon.nickname` after calling `.destroy`)